ophirofox/content_scripts/config.js

```diff
@@ -98,6 +98,13 @@ const ophirofox_config_list = [
     AUTH_URL:
       "http://scd1.univ-fcomte.fr/login?url=https://nouveau.europresse.com/access/ip/default.aspx?un=FCOMTET_1",
   },
+  {
+    name: "Université de Pau et des Pays de l'Adour",
+    domains: ["univ-pau.fr"],
+    LOGIN_URL: "https://sso.univ-pau.fr/cas/login",
```

Comment:
Ici est-ce que ce ne serait pas plutôt

```suggestion
    LOGIN_URL: "https://rproxy.univ-pau.fr/login",
```